tools/greenbone-nvt-sync.in

```diff
@@ -88,9 +88,22 @@ ENABLED=1
 REFRESH_ONLY=0
 
 # LOG_CMD defines the command to use for logging. To have logger log to stderr
-# as well as syslog, add "-s" here.
+# as well as syslog, add "-s" here. The logging facility is check. In case of error
+# all will be logged in the standard error and the socket error check will be
+# disabled.
 LOG_CMD="logger -t $SCRIPT_NAME"
 
+check_logger () {
+  logger --socket-error=on -p daemon.info -t $SCRIPT_NAME "Checking logger" --no-act
```

Comment:
The output of this test command could be redirected to avoid printing `logger: socket /dev/log: No such file or directory` or similar messages to the console.